modules/touch/include/touchinteraction.h

```diff
@@ -67,13 +66,11 @@ class TouchInteraction : public properties::PropertyOwner {
 public:
     TouchInteraction();
 
-    // for interpretInteraction()
-    enum Type {
+    enum InteractionType {
```

Comment:
In our [coding style](https://github.com/OpenSpace/OpenSpace/wiki/Coding-Style#8-enumerations-must-not-be-in-upper-case-and-should-be-strongly-typed), it says that enums should be "_strongly typed_" so this would become `enum class InteractionType`. But I'm not sure how strongly we keep to this guideline.

Reply:
:+1: for strongly typing it.  It prevents a bunch of potential errors and confusions

Reply:
👍